source/openapi-admin-v3.yaml

```diff
@@ -7221,6 +7242,18 @@ components:
           type: object
           description: |-
             **Deprecated**
+        indexed_queryable_fields_names:
+          type: array
+          description: |-
+            A list of field names to use as the indexed queryable field. While this property is an array, Sync 
+            currently supports only one one indexed queryable field. Therefore, this array may contain at most 
+            one element.
+
+            The indexed queryable field must be present in the schema and be the same eligible field type in 
+            every collection you sync. The indexed queryable field name must also appear in queryable_fields_names 
+            since this is a global queryable field.
+          items:
+            type: string
```

Comment:
Seems like we are also missing asymmetric_tables as an option

Reply:
Added!